modules/local/hifiadapterfilt/main.nf

```diff
@@ -0,0 +1,55 @@
+process HIFIADAPTERFILT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    container 'quay.io/microbiome-informatics/hifiadapterfilt:v1.0.1'
+
+    containerOptions {
+        if (workflow.containerEngine == 'singularity') {
+            return "--bind ${moduleDir}/tests/fixtures/:/data/"
+        } else {
+            return "-v ${moduleDir}/tests/fixtures/:/data/"
+        }
+    }
```

Comment:
is this needed at all? it sounds like you're mounting the unit test fixtures anytime this module gets run

Reply:
Luckily hifiadapterfilt has been accepted in bioconda, this will change soon! :)